test/unit/cluster-list-test.js

```diff
@@ -0,0 +1,62 @@
+describe("Unit: List Clusters", function () {
```

Comment:
We should not mention specifically name as "list" in file name as well in describe block . File name should be "cluster-controller-test.js" and inside describe should be "UNIT CONTROLLER - clusterController".